### ISSUE TITLE
"Operation complete" dialog box

### ISSUE THREAD

**Issue by marangonico** [nofaceinbook/muxp#18]
Display a message box after the completion of the process.

**Comment by nofaceinbook** [comment]
Yes good point, but only if you start muxp not from command line (meaning to hand over the .muxp file as first argument). The message box will allow to either quit or restart with a new process.
